code/modules/mob/living/carbon/human/life/handle_disabilities.dm

```diff
@@ -60,6 +60,7 @@
 						"SARAH HALE DID IT!!!", \
 						"Don't tell Chase", \
 						"not so tough now huh", \
+						"IF YOU FAGGETS DONT LIKE THE PEEARES WHY DONT YOU CODE YER OWN?!", \
```

Comment:
pee arrs?

Reply:
PRs.

Reply:
Pull requests

Reply:
i mean it literally
pee arrs or PEEARES
i feel the former is better

Reply:
pears